src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
> In the meanwhile I can open an issue on pecl-event to let the author know about this.

Yes, pushing this upstream is where this should be addressed in the first place! :+1: 

> I'd say it's better to have additional warnings than suppressing everything.
> […]
> 3. Remove the @-suppression and document the change and it's impact in the changelog

Absolutely agree here! :+1:

Given this only triggers when an uncaught warning or exception is passed through, I expect this to have very little impact in actual consumer code. After all, this is something that *should* never happen in the first place. Also, this only triggers an *additional* warning, so I agree this should be safe to add regardless :+1: 

This means that even if upstream takes longer to fix this and this be inconsistent across different upstream versions, I think it makes perfect sense to get this in now already :+1: